frontend/components/token-classifier/results/TextSpansStatic.vue

```diff
@@ -17,14 +17,25 @@
 
 <template>
   <div v-if="textSpans.length" ref="list" class="content__input">
-    <text-span-static
-      v-once
-      v-for="(token, i) in textSpans"
-      :key="i"
-      :record="record"
-      :token="token"
-      :dataset="dataset"
-    />
+    <template v-if="dataset.query.score">
```

Comment:
If we move this logic to the frontend/components/token-classifier/results/RecordTokenClassificationAnnotation.vue and frontend/components/token-classifier/results/RecordTokenClassificationExploration.vue we could keep intact this component. Also, we can gain performance if no score filter is applied.